test/versioned/amqplib/amqp-utils.js

```diff
@@ -194,26 +190,18 @@ function verifyProduce(t, tx, exchangeName, routingKey) {
     ]
     // 0.9.0 flattened the segment tree
     // See: https://github.com/amqp-node/amqplib/pull/635/files
-  } else if (semver.gte(pkgVersion, '0.9.0')) {
+  } else {
```

Comment:
I think(?) this comment was tied to the semver based conditional. Should it be removed?